### PR TITLE
Fix the bug on demo player

### LINKED ISSUE #131: Demo player is sometime display wrong on linux and macOS with chromium-based browser
CleanShot 2564-11-06 at 19 46 36@2x

(Test on Chrome, MacBook Pro)

The demo player card sometime not correctly display on linux and macOS device with chromium-based browser. (It's show like that but it's still functional)

### LINKED ISSUE #134: Demo player on beatmap card improvement
This demo player on beatmap card is using the very old version of JQuery. It's need to be updated. If it's improved I think it's should fix the proflem like #131 too.

### PR DESCRIPTION
**Pull request by HelloYeew**
I try to change the version of JQuery. Need to try on the device that the bug occured.

### CODE REVIEW

**Review by SainTurDaY27 — APPROVED** [review]

**Review by some-random-idiot — APPROVED** [review]

File: beatmap_collections/templates/beatmap_collections/add_beatmap.html [hunk 1]
```diff
@@ -109,6 +109,9 @@ <h2>Suggest Beatmap</h2>
 {% block js %}
 <script src="https://code.jquery.com/jquery-3.6.0.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script>
 <script src="{% static "js/live_beatmap_card.js" %}"></script>
-<script src="https://code.jquery.com/jquery.min.js"></script>
+<script
+        src="https://code.jquery.com/jquery-1.11.2.min.js"
+        integrity="sha256-Ls0pXSlb7AYs7evhd+VLnWsZ/AqEHcXBeMZUycz/CcA="
+        crossorigin="anonymous"></script>
 <script src="{% static "js/player.js" %}"></script>
```
Comment (pontakornth) [inline beatmap_collections/templates/beatmap_collections/add_beatmap.html:116]: This line has multiple jQuery. If the lower one is used, can you remove the upper one?

**Review by pontakornth — CHANGES_REQUESTED** [review]

**Review by Siraphop4Nene — APPROVED** [review]
Nice work KUB!!!!

File: beatmap_collections/templates/beatmap_collections/add_beatmap.html (hunk 1 above)
Comment (HelloYeew) [reply beatmap_collections/templates/beatmap_collections/add_beatmap.html]: I tried and the live demo can use the version that demo player need. Removed.

**Review by pontakornth — APPROVED** [review]
mocha_stonk

**Comment by HelloYeew** [comment]
Let's merge the rabbit!

**Pull request merged**